packages/nimbus/src/components/radio-input/radio-input.stories.tsx

```diff
@@ -0,0 +1,445 @@
+        <RadioInput.Option value="yes" key="input5">
+          Yes
+        </RadioInput.Option>
+        <RadioInput.Option value="no" key="input6">
+          No
+        </RadioInput.Option>
+        <RadioInput.Option value="probably" key="input7">
+          Probably
+        </RadioInput.Option>
+        <RadioInput.Option value="meh" key="input8">
+          Meh
+        </RadioInput.Option>
+      </RadioInput.Root>
+
+      <hr />
+      {/* Vertical */}
+      <RadioInput.Root
+        name="storybook-radio-grouping-direction-vertical"
+        orientation="vertical"
+      >
+        <RadioInput.Option value="ja" key="input1-vert">
+          Ja
+        </RadioInput.Option>
+        <RadioInput.Option value="nein " key="input2-vert">
+          Nein
+        </RadioInput.Option>
+        <RadioInput.Option value="wahrscheinlich" key="input3-vert">
+          Wahrscheinlich
+        </RadioInput.Option>
+        <RadioInput.Option value="naja" key="input4-vert">
+          Naja
+        </RadioInput.Option>
+      </RadioInput.Root>
+    </Stack>
+  ),
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+    const radioInputs = [
+      canvas.getByLabelText("Yes") as HTMLInputElement,
+      canvas.getByLabelText("No") as HTMLInputElement,
+      canvas.getByLabelText("Probably") as HTMLInputElement,
+      canvas.getByLabelText("Meh") as HTMLInputElement,
+    ];
+
+    for (const [i, radio] of radioInputs.entries()) {
+      await step(
+        `Select radioInput ${i + 1} and assert only it is selected`,
+        async () => {
+          await userEvent.click(radio);
+          const checkedRadioInputs = radioInputs.filter((r) => r.checked);
+          await expect(checkedRadioInputs).toHaveLength(1);
+          await expect(checkedRadioInputs[0]).toBe(radio);
+        }
+      );
+    }
+  },
+};
+
+export const WithFormField: StoryObj = {
+  render: () => (
+    <FormField.Root>
+      <FormField.Label>Favorite Artist</FormField.Label>
+      <FormField.Input>
+        <RadioInput.Root name="artist">
+          <RadioInput.Option value="dreamy">Dreamy Dave</RadioInput.Option>
+          <RadioInput.Option value="cure">The Cure</RadioInput.Option>
+          <RadioInput.Option value="gaga">Lady Gaga</RadioInput.Option>
+        </RadioInput.Root>
+      </FormField.Input>
+      <FormField.Description>Pick your favorite artist.</FormField.Description>
+    </FormField.Root>
+  ),
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+
+    // FormField elements
+    const formLabel = canvas.getByText("Favorite Artist");
+
+    // RadioInput elements
+    const radioGroup = canvas.getByRole("radiogroup");
+    const radioDreamyDave = canvas.getByLabelText("Dreamy Dave");
+    const radioTheCure = canvas.getByLabelText("The Cure");
+    const radioLadyGaga = canvas.getByLabelText("Lady Gaga");
```

Comment:
I don't know if this is a big deal or not, but the name "Lady Gaga" is trademarked and since our sb is public, I'd suggest we change it 🙃